handsontable/src/core/focusCatcher/index.js

```diff
@@ -122,9 +127,15 @@ export function installFocusCatcher(hot) {
                 ? getMostTopStartPosition(hot) : getMostBottomEndPosition(hot);
             }
 
+            const result = hot.runHooks('modifyUnfocusOnTabNavigation', event.shiftKey ? 'to_above' : 'to_below');
```

Comment:
I'm not sure about the hook name. The hook is triggered only when the internal element (table) loses its focus (and passes to the Pagination, Dialog, or other plugin in the feature). WDYT about changing the name to `focusTableTransfer`, or `tableFocusExit`. I'd not use the `modify` phrase here, as there is nothing to modify, like in the `modifyFocusOnTabNavigation` hook, where you can modify which cell can be selected after table focus.

@jansiegel WDYT?